src/bacnet/basic/object/sc_netport.c

```diff
@@ -553,6 +586,16 @@ bool Network_Port_SC_Hub_Function_Enable_Set(uint32_t object_instance,
         return false;
     params->SC_Hub_Function_Enable = value;
 
+    return true;
```

Comment:
@Michail-Antropov This difference shows that Network_Port_Changes_Pending_Set() was dropped from the definition of Network_Port_SC_Primary_Hub_URI_Set(). Is that intentional?